sdk/lib/crt/misc/dbgrpt.cpp

```diff
@@ -16,6 +16,8 @@
 #include <windows.h>
 
 #undef OutputDebugString
+#undef _CrtSetReportMode
+#undef _CrtSetReportFile
```

Comment:
Are those two undefs necessary?

Reply:
This source is being compiled even on Release time. On Release time, these macros will be defined to disable the debug functions.